pkg/kv/kvserver/loqrecovery/server.go

```diff
@@ -73,6 +85,7 @@ func NewServer(
 	rpcCtx *rpc.Context,
 	knobs base.ModuleTestingKnobs,
 	decommission func(context.Context, roachpb.NodeID) error,
+	decommissionStatusFn func(context.Context, roachpb.NodeID) (livenesspb.MembershipStatus, bool),
```

Comment:
Good catch! Since it is all happening inside Verify, we can get the necessary info from the IsLiveMap and remove status function. We will also have no dependency on liveness and only depend on livenesspb for the map.